Lib/test/test_string_literals.py

```diff
@@ -118,8 +118,7 @@ def test_eval_str_invalid_escape(self):
             eval("'''\n\\z'''")
         self.assertEqual(len(w), 1)
         self.assertEqual(w[0].filename, '<string>')
-        if use_old_parser():
-            self.assertEqual(w[0].lineno, 1)
+        self.assertEqual(w[0].lineno, 1)
```

Comment:
No assert for column?

Reply:
We cannot do this here, because `WarningMessage` objects do not hold column information. Added an assertion a bit further down, where a `SyntaxError` is thrown.